exporter/awscloudwatchlogsexporter/README.md

```diff
@@ -27,6 +27,8 @@ exporters:
   awscloudwatchlogs:
     log_group_name: "testing-logs"
     log_stream_name: "testing-integrations-stream"
+    sending_queue:
+      queue_size: 50
```

Comment:
Just for the record, I don't know if this is a good value here, so if anyone has an idea, please let me know or open another PR to update this.

Reply:
I've now fixed defaulting and will rely on `exporterhelper.DefaultQueueSettings().QueueSize`, so user doesn't have to think of a number :)